scripts/hooks/pre-commit.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# lint modified go files
+golangci-lint run --fix --new -c .golangci.yml
```

Comment:
This can be too slow.. Can we only enable `fast` linters?

Reply:
Confusing here, did you mean that we should only lint the changed go files, not all *.go files?

Or did you mean that we should remove `-c .golangci.yml`?

Reply:
```
--fast                           Run only fast linters from enabled linters set (first run won't be fast)
```